apps/avifenc.c

```diff
@@ -1442,6 +1440,8 @@ int main(int argc, char * argv[])
     avifImage ** gridCells = NULL;
     avifImage * gridSplitImage = NULL; // used for cleanup tracking
 
+    INIT_ARGV()
```

Comment:
I suggest we call `INIT_ARGV()` as early as possible in the main() function.

At least, we should try to call `setlocale()` as early as possible in the main() function.

Reply:
Indeed: I wanted to use the cleanup goto by default but I'll free things by hand then. Done